packages/ifpack2/src/Ifpack2_BlockTriDiContainer_impl.hpp

```diff
@@ -1432,17 +1448,27 @@ namespace Ifpack2 {
     };
     
 #if defined(KOKKOS_ENABLE_CUDA) 
+    int ExtractAndFactorizeRecommendedCudaTeamSize(const int blksize,
```

Comment:
This function is a candidate for `constexpr` with C++14: https://en.cppreference.com/w/cpp/language/constexpr